tests/Microsoft.Identity.Test.Unit/TokenClientTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Client.OAuth2;
+using Microsoft.Identity.Client.Utils;
+using Microsoft.Identity.Test.Common.Core.Helpers;
+using Microsoft.Identity.Test.Common.Core.Mocks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium.Remote;
+
+namespace Microsoft.Identity.Test.Unit
+{
+    [TestClass]
+    public class TokenClientTests : TestBase
+    {
+        [TestMethod]
+        public async Task SendTokenRequest_MissingAccessTokenTypeInResponse_Throws_Async()
```

Comment:
I would move this to [here](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/0d53f3e80e29f9c6d172e069c10e02d71e24f65e/tests/Microsoft.Identity.Test.Unit/PublicApiTests/AuthenticationSchemeTests.cs#L93)

Reply:
👍